app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,129 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+
+ *= require_tree .
+ *= require_self
+ *= require foundation_and_overrides
+ *= require_tree ./modules
+ */
+
+ @font-face {
+   font-family: "Nexa Light";
+   src: font-url('nexa_light.otf');
+ }
+ .larger {
+    font-size: 150%;
+    color: white;
+ }
+
+ .smaller {
+    font-size: 50%;
+ }
+
+ .tab {
+    text-indent: 50px;
+ }
+
+ .review {
+    border-style: solid;
+    border-width: 5px;
+    border-radius: 10px;
+    border-color: #858585;
+    padding-left: 50px;
+ }
+
+ .vote-arrows {
+  width: 40px;
+  text-align: center;
+}
+
+.inline {
+  display: inline;
+}
+
+
+#wrapper {
+  background-image: url("http://imgur.com/7OV7GsB.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  width: 100% !important;
+  z-index: 0;
+}
+
+#featured {
+  position: relative;
+  top: 65px;
+}
+
+#sidebar_img {
+  width: 300px;
+  height: 465px;
+}
+
+body {
+  background: #222222;
+  font-family: 'Nexa Light', sans-serif;
+  color: white;
+}
+
+#title {
+  background: #222222;
+  text-align: center;
+}
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Nexa Light', sans-serif;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line